client.go

```diff
@@ -24,12 +24,12 @@ import (
 
 // Client does the work of perform the REST requests to the TVDB api endpoints.
 // With its methods you can run almost all the requests provided in the TVDB
-// api.
+// api.  It is engough to specify the Apikey, if you leave the Userkey and Username as nil.
 type Client struct {
-	// The TVDB API key, User key, User name. You can get them here http://thetvdb.com/?tab=apiregister
+	// The TVDB API key, User key, Username. You can get them here http://thetvdb.com/?tab=apiregister
 	Apikey   string
-	Userkey  string
-	Username string
```

Comment:
I would prefer type safety and keep the type `string` instead of `interface{}` and pass `null` in the login request if the string is empty (`""`).